api/Utilities/UserUtilities/UserUtilities.cs

```diff
@@ -56,6 +56,7 @@ public async Task<string> CreateToken(UserLoginDto userDto)
             List<Claim> claims = new List<Claim>
             {
                 new Claim(ClaimTypes.Email, userDto.Email),
+                new Claim(ClaimTypes.NameIdentifier, user.Id.ToString()),
             };
 
             if (userRoles.Any(ur => ur.UserRole == UserRoles.Moderator || ur.UserRole == UserRoles.Administrator || ur.UserRole == UserRoles.User))
```

Comment:
Nie tego tyczy PR, ale dlaczego jak user ma role Administrator to dajemy mu role User? Wiem że niżej jest to nadpisywane, ale jestem ciekaw czy to celowe

Reply:
Nie mogę zaznaczyć kodu poniżej, ale ogólnie jeśli się nie mylę, w linii 77-79 czytasz ten secret z pliku, który jest w repo, także każdy możę ten secret odczytać i zmieniać sygnaturę JWT.

W skrócie mogę zmienić sobie rolę na Admin i podać token do localstorage czy tam postmanie i mieć inne uprawnienia

Reply:
Tak, dodaje role Usera też Adminowi, żeby miał do wszystkiego pełne prawa i nie trzeba było pisać tego ręczenie w enpointach.

A z tym secretem z pliku to jest na razie tak dewelopersko, bo spodziewam się że na produkcji byśmy to chcieli jakoś bardziej zabezpieczyć.